types/simple-element-resize-detector/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for simple-element-resize-detector 1.3
+// Project: https://github.com/developit/simple-element-resize-detector#readme
+// Definitions by: Nathan Bierema <https://github.com/Methuselah96>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function resizeDetector(
+    element: HTMLElement,
+    handler: () => void
```

Comment:
```suggestion
    handler: (element: HTMLElement) => void
```
it seems to pass back the reference to element being observed:
https://github.com/developit/simple-element-resize-detector/blob/master/index.js#L10